GiniVisionExample/GiniVisionExample/ViewControllers/HelpViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  HelpViewController.swift
+//  GiniVisionExample
+//
+//  Created by Enrique del Pozo Gómez on 11/21/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import UIKit
+import GiniVision
+
+typealias HelpLink = (title: String, url: URL?)
+typealias HelpVersion = (title: String, version: String)
```

Comment:
Tuples here are an interesting choice. Usually I'd say that using them in such a way is a shortcut and wouldn't recommend it. But here it seems like it's working out. I'm ok with keeping it, but I think it's something you should be looking out for - it's easy to misuse them IMHO.

Reply:
I use them here since IMO they are not models. I thought about creating a HelpItem protocol and make them implement it but I thought that it would be an overkill. 
I didn't want to make it to complicated but since I had to start decoupling one view controller into a view and a view model maybe it makes sense to decoupling this class too in a future PR.